flight/targets/aq32/fw/pios_config.h

```diff
@@ -32,6 +32,7 @@
 #include <pios_flight_config.h>
 
 /* Major features */
+#define PIOS_INCLUDE_LQG
```

Comment:
This isn't really a PIOS_INCLUDE because it's not in PIOS right?

We typically have like

```
/* Flags that alter behaviors */

#define SYSTEMMOD_RGBLED_SUPPORT
#define SYSTEMMOD_RGBLED_VIDEO_HACK

#define CAMERASTAB_POI_MODE
```

But I question whether there should be a flag at all, since you're turning it on, on all targets, and it's something that people will expect to work.

Reply:
Yep, should be STABILIZATION_LQG or something if it does stay.

Reply:
I put some defines there, to exclude some targets. I still wasn't sure the F3 can run it. The F3s can easily reach 80% as is according to the guys on IRC. I can rename the define for it to be proper.

Reply:
Also, one F3 target was pretty damn low on memory. It was ufo's F3. Some OmnibusF3 clone with CharOSD I think. Dropping the LQGSolver task might have improved that situation though.

Reply:
We have also done half of the inittask thing since then.